polylogue/cli/app.py

```
@@ -158,19 +210,45 @@ def run_help_cli(args: argparse.Namespace, env: CommandEnv) -> None:
             available = ", ".join(sorted(choices)) or "<none>"
             console.print(f"Available commands: {available}")
-            return
+            raise SystemExit(1)
+
         console.print(f"[cyan]polylogue {topic}[/cyan]")
         subparser.print_help()
+
+        # Show all examples for this command
+        examples = COMMAND_EXAMPLES.get(topic, [])
+        if examples:
+            console.print(f"\n[bold cyan]EXAMPLES[/bold cyan]\n")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)